.gitignore

```diff
@@ -11,3 +11,6 @@ build/
 .project
 .classpath
 .vscode/
+.idea/
+**/.editorconfig
```

Comment:
:question: is editor config ignore intended?

Reply:
For now, yes, it's intended - we should probably match what the primary openEQUELLA repo is doing for styling (google java)